addons/binding/org.openhab.binding.gpstracker/README.md

```diff
@@ -152,6 +175,123 @@ sitemap gpstracker label="GPSTracker Binding" {
     Text item=atHomeEX
     Text item=lastSeenEX
     Text item=batteryEX
+    Text item=accurayEX
```

Comment:
We have the spelling mistake here too: accuracyEX

Reply:
Sorry